includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -1455,7 +1458,7 @@ public function process_payment( $order_id ) {
 					'result'   => 'success',
 					'redirect' => $redirect,
 				),
-				( $doing_classic_payment ? array( 'amazonCreateCheckoutParams' => wp_json_encode( $create_checkout_config ) ) : array() )
+				( $doing_classic_payment ? array( 'amazonCreateCheckoutParams' => wp_json_encode( $create_checkout_config ), 'amazonEstimatedOrderAmount' => self::get_estimated_order_amount() ) : array() )
```

Comment:
I didn't fully get why are we sending the estimated order amount as a separate parameter? Is it because there are cases where is no needed it?

Reply:
The reason is that amazonCreateCheckoutParams is required for the Amazon Checkout to initiate. estimatedOrderAmount isn't. On the JS, we are checking for their existense individually and they are being used in different properties of the settings object used to initiate an Amazon Checkout. Also i wanted to avoid any case of introducing a regression to the Classic gateway.